lms/templates/courseware/progress.html

```diff
@@ -154,22 +154,24 @@ <h3 class="hd hd-3" id="chapter_${loop.index}">${ chapter['display_name']}</h3>
                                 %for section in chapter['sections']:
                                     <div>
                                         <%
-                                        earned = section.all_total.earned
-                                        total = section.all_total.possible
-                                        percentageString = "{0:.0%}".format( float(earned)/total) if earned > 0 and total > 0 else ""
+                                        subsection_grade_long = ""
+                                        subsection_grade_short = ""
+                                        if section.show_grades(staff_access):
+                                          earned = section.all_total.earned
+                                          total = section.all_total.possible
+                                          percentageString = ''
```

Comment:
Oops, missed that one.. Fixed with https://github.com/edx/edx-platform/pull/15018/commits/cc37c0fbce1ac195717ac38723a3510021968308#diff-54c933c75934e410ed718cbb2345158aR162